### DISCUSSION TITLE
Princess Zelda Fabric and Gerudo-king Fabric

### DISCUSSION THREAD

**Discussion by 465489886** [jordanbtucker/totk-no-loot-boxes#20] [Q&A]
Can I get Princess Zelda Fabric and Gerudo-king Fabric without this mod

**Comment by jordanbtucker** [comment]
I'm not sure yet. I'll let you know if I find out.

**Reply by 465489886** [reply]
At least not obtainable from the BotW Zelda amiibo or the Smash Bros. Ganondorf amiibo，right?

**Reply by jordanbtucker** [reply]
Correct.

**Reply by 465489886** [reply]
thx~
